lk2nd/display/panel/generated/lk_panel_smd_549_1080p_alb_cmd_v0.h

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2025 FIXME
+// Generated with linux-mdss-dsi-panel-driver-generator from vendor device tree:
+//   Copyright (c) 2014, The Linux Foundation. All rights reserved. (FIXME)
+
+#ifndef _PANEL_SMD_549_1080P_ALB_CMD_V0_H_
+#define _PANEL_SMD_549_1080P_ALB_CMD_V0_H_
```

Comment:
are you using lk1st or otherwise using this panel driver? if so, I'd expect you document the device as quirky in the dts and provide notes on how to enable the panel driver.